### PR TITLE
Feature/gcc 9.3

### PR DESCRIPTION
**Pull request by KVS85**
devel PR: https://github.com/arangodb/arangodb/pull/11476

### CODE REVIEW

**Comment by KVS85** [comment]
http://jenkins.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/9548/

**Comment by KVS85** [comment]
http://jenkins.arangodb.biz:8080/job/arangodb-matrix-pr-windows/3429/

**Comment by KVS85** [comment]
Tests blue.

File: containers/buildAlpine4.docker/tools/install-openldap.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+cd openldap-$OPENLDAPVERSION
+# cp -a /tools/config.* ./build
+CPPFLAGS=-I/opt/openssl-$OPENSSLPATH/include \
+LDFLAGS=-L/opt/openssl-$OPENSSLPATH/lib \
+  ./configure --prefix=/opt/openssl-$OPENSSLPATH --with-threads --with-tls=openssl --enable-static --disable-shared
+make depend && make -j64
+make install
+cd /tmp
+rm -rf openldap-$OPENLDAPVERSION.tgz openldap-$OPENLDAPVERSION
+
```
Comment (joerg84) [inline containers/buildAlpine4.docker/tools/install-openldap.sh:22]: Trailing blank

**Pull request merged**